charts/datadog/templates/_helpers.tpl

```diff
@@ -893,3 +893,16 @@ Create RBACs for custom resources
     false
   {{- end -}}
 {{- end -}}
+
+{{/*
+Return all namespaces with disabled Single Step Instrumentation
+*/}}
+{{- define "apmInstrumentation.disabledNamespaces" -}}
```

Comment:
The main reason not to allow `disabledNamespaces` and `enabledNamespaces` set at the same time is to prevent unspecified behavior when a namespace is present in both lists.

`disabledNamespaces` shouldn't be empty only when SSI is on for the cluster and we just want disable it in the few namespaces.

If users set enabledNamespaces, generally it means that SSI is disable in the whole cluster.